hetu-transport/src/main/java/io/hetu/core/transport/execution/buffer/PagesSerdeUtil.java

```diff
@@ -155,21 +165,38 @@ public static long writePages(PagesSerde serde, SliceOutput sliceOutput, Iterato
     {
         private final PagesSerde serde;
         private final SliceInput input;
+        private final int prefetchSize;
+        List<Page> prefetchedPages;
 
         PageReader(PagesSerde serde, SliceInput input)
+        {
+            this(serde, input, 1);
```

Comment:
intead of 1 define constant as NO_PREFETCH=1 and use that

Reply:
addressed